src/syntax/runtime/parser.rs

```diff
@@ -90,15 +90,16 @@ pub fn parse(source: &str) -> Result<ast::Resource> {
 
     let mut entries = vec![];
 
-    let entry = get_entity(&mut ps)?;
+    let message = get_entity(&mut ps)?;
+    let entry = ast::Entry::Message(message);
 
     entries.push(entry);
```

Comment:
Can you inline the entry value into `push` please? No need for a separate variable that is used only once in the next line.